README.md

```diff
@@ -5,13 +5,16 @@ The Ballerina standard library
 
 |Level| Modules | Latest Version | Build | Last Commit | Open Issues | Open Pull Requests |
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|0|[java.arrays](https://github.com/ballerina-platform/module-ballerina-java.arrays)| [![GitHub Release](https://img.shields.io/github/release/ballerina-platform/module-ballerina-java.arrays.svg?label=)](https://github.com/ballerina-platform/module-ballerina-java.arrays/releases)| [![Build](https://github.com/ballerina-platform/module-ballerina-java.arrays/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-java.arrays/actions?query=workflow%3ABuild)| [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-java.arrays.svg?label=)](https://github.com/ballerina-platform/module-ballerina-java.arrays/commits/master)| [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/java.arrays.svg?label=)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fjava.arrays) | [![GitHub pull-requests](https://img.shields.io/github/issues-pr/ballerina-platform/module-ballerina-java.arrays.svg?label=)](https://github.com/ballerina-platform/module-ballerina-java.arrays/pulls)|
```

Comment:
Let's start the level from 1. No point of zero isnt it?

Reply:
That means completely remove level 0, OR push level zero to the bottom?.

Reply:
Lets mark Level 0 modules as 1

Reply:
+1, and fixed in here https://github.com/ballerina-platform/ballerina-standard-library/pull/193/commits/39bcac6a9bdf7f3b64604d756371ff566a91623e